gapis/replay/interfaces.go

```diff
@@ -72,3 +81,9 @@ type Issue struct {
 	Severity service.Severity // The severity of the issue.
 	Error    error            // The issue's error.
 }
+
+type Timestamps struct {
```

Comment:
Document string please. Especially the units of the Time. Even better would be using a time.Duration here instead, that way the units are baked in.

Reply:
done